src/Rule/MaxLength.php

```
@@ -13,9 +13,18 @@ class MaxLength implements ValidationRuleInterface
      */
     private $length;
 
-    public function __construct(int $length)
+    /**
```

This file should not be part of the PR.
Although seeing, this it makes me think that a negative integer should not be passed to the function, but it is currently possible.

Could you make it throw an InvalidArgumentException in that case?